### DISCUSSION TITLE
Warum eigenes Addon?

### DISCUSSION THREAD

**Discussion by AWqxKAWERbXo** [FriendsOfREDAXO/yform_ui#3] [General]
Es gibt bereits ein FOR-Addon für die Verbesserung der usability / Modifikation der ui. Warum nicht als Plugin dafür veröffentlichen?

**Comment by eaCe** [comment]
Hatte darüber nachgedacht, mich dann dagegen entschieden da ich der Meinung war das es nicht in die usability passt.
Aber ich kann das auch da rein werfen. Ich frage mal bei den Entwicklern von yform_usability nach.

**Comment by AWqxKAWERbXo** [comment]
Cool, das Diskussions-Feature kenne ich noch gar nicht. Vielleicht kann man sich auf Slack und beim Lead ein Meinungsbild einholen.

**Reply by eaCe** [reply]
Finde es besser für solche Fragen, ist ja weniger ein konkretes Problem :)

**Reply by eaCe** [reply]
>  YForm usability wird zu selten aktualisiert. Das könnte Deine Entwicklung ausbremsen, denke ich. Aktuell ist usability nicht mit YForm 4 kompatibel. Da hakt es gewaltig

Thomas meint das wäre aktuell keine gute Idee

**Reply by AWqxKAWERbXo** [reply]
yform_usability ist inzwischen YForm-4-kompatibel. Und wird jetzt automatisch aktualisiert, wenn ein Release auf GitHub erfolgt.
